nicegui/run_with.py

```diff
@@ -27,4 +28,4 @@ def run_with(
     app.on_event('startup')(lambda: handle_startup(with_welcome_message=False))
     app.on_event('shutdown')(lambda: handle_shutdown())
```

Comment:
Maybe this way could work as well?

```suggestion
    app.on_event('shutdown')(handle_shutdown)
```

Reply:
You're probably right. I think the lambda statement was just for symmetry with the line above.